src/Exception/EntryDoesNotExistException.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace TomPHP\ConfigServiceProvider\Exception;
+
+final class EntryDoesNotExistException extends RuntimeException
```

Comment:
@tomphp 

Not sure, should this be 

```diff
-final class EntryDoesNotExistException extends RuntimeException
+final class EntryDoesNotExistException extends \RuntimeException
```

instead?

On the other hand, do we want to be consistent in regard to importing classes from the root namespace?

Personally, I used to import them all the time, now I don't.

I guess it doesn't matter match, but what do you think about being consistent?